aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -320,7 +321,13 @@ class TypedOperatorHandle<Return (Args...)> final : public OperatorHandle {
   TypedOperatorHandle& operator=(const TypedOperatorHandle&) = default;
 
   Return call(Args... args) const {
-    return c10::Dispatcher::singleton().call<Return, Args...>(*this, std::forward<Args>(args)...);
+    detail::unused_arg_(args...);  // workaround for a false-positive warning about unused parameters in gcc 5
+    auto dispatchKey = operatorIterator_->op.dispatchKeyExtractor()
+      .template getDispatchKeyUnboxed<Args...>(
+        DispatchKeySet::FULL,
+        args...
+    );
+    return c10::Dispatcher::singleton().callWithDispatchKey<Return, Args...>(*this, dispatchKey, args...);
```

Comment:
call() and callWithDispatchKey() both have duplicate functions (one within this TypedOperatorHandle class and one  as part of the Dispatcher singleton), but callWithDispatchKey is longer and called in two places so I opted not to inline it.